tests/Feature/RegistationTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace Tests\Feature;
+
+use Mockery;
+use Tests\TestCase;
+use App\Event;
+use App\Participant;
+use App\Helpers\Payment\PaymentInterface;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use App\Helpers\Payment\MolliePaymentProvider;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\DB;
+
+
+
+class RegistationTest extends TestCase
+{
+    private static function clearDB()
```

Comment:
Laravel biedt een trait genaamd `RefreshDatabase` die je database ververst na afloop van een test. Is misschien een nettere oplossing dan dit? Zie https://laravel.com/docs/5.8/database-testing#resetting-the-database-after-each-test

Reply:
De RefreshDatabase  voert niet je seedings uit, dus dat werkt niet voor die payment tests, vandaar dat ik zo moeilijk doe. Ik heb voor als nog geen trait of iets gevonden wat bij elke test je seedings uitvoert, wel kwam ik wat tegen die met RefreshDatabase  een "backup" file van je database in een sqlLite database duwde speciaal voor testing, maar dat vereist zoveel meer werk, ik dacht dat kan later ook .